docs/source/openshift/openshift-client-binary.adoc

```diff
@@ -46,30 +39,25 @@ for example, logging into another OpenShift instance, you can run:
 $ oc config use-context minishift
 ----
 
-For an introduction to `oc` usage, see the
-https://docs.openshift.com/enterprise/3.2/cli_reference/get_started_cli.html[Get Started with the CLI]
-topic in the OpenShift documentation.
+For an introduction to `oc` usage, see the https://docs.openshift.com/enterprise/3.2/cli_reference/get_started_cli.html[Get Started with the CLI] topic in the OpenShift documentation.
 
 [[log-into-cluster]]
-=== Logging into the cluster
+== Logging into the cluster
 
-By default, _cluster up_ uses
-https://docs.openshift.org/latest/install_config/configuring_authentication.html#AllowAllPasswordIdentityProvider[AllowAllPasswordIdentityProvider]
-to authenticate against the local cluster. This means any non-empty user name and password can
-be used to login to the local cluster. The recommended user name and password combination is
-developer/developer, because it already has a default project _myproject_ and also can
-https://docs.openshift.org/latest/architecture/additional_concepts/authentication.html#authentication-impersonation[impersonate]
-the administrator. This allows you to run administrator commands using the `--as system:admin` parameter.
+By default, _cluster up_ uses https://docs.openshift.org/latest/install_config/configuring_authentication.html#AllowAllPasswordIdentityProvider[AllowAllPasswordIdentityProvider] to authenticate against the local cluster.
+This means any non-empty user name and password can be used to login to the local cluster.
+The recommended user name and password combination is developer/developer, because it already has a default project _myproject_ and also can https://docs.openshift.org/latest/architecture/additional_concepts/authentication.html#authentication-impersonation[impersonate] the administrator.
```

Comment:
The recommended user name and password combination is _developer_ and _developer_ respectively since it already has a default project ...